bin/hbase

```diff
@@ -775,25 +783,31 @@ HBASE_OPTS="$HBASE_OPTS -Dhbase.log.dir=$HBASE_LOG_DIR"
 HBASE_OPTS="$HBASE_OPTS -Dhbase.log.file=$HBASE_LOGFILE"
 HBASE_OPTS="$HBASE_OPTS -Dhbase.home.dir=$HBASE_HOME"
 HBASE_OPTS="$HBASE_OPTS -Dhbase.id.str=$HBASE_IDENT_STRING"
-HBASE_OPTS="$HBASE_OPTS -Dhbase.root.logger=${HBASE_ROOT_LOGGER:-INFO,console}"
+HBASE_OPTS="$HBASE_OPTS -Dhbase.root.logger.level=${HBASE_ROOT_LOGGER_LEVEL:-INFO}"
+HBASE_OPTS="$HBASE_OPTS -Dhbase.root.logger.appender=${HBASE_ROOT_LOGGER_APPENDER:-console}"
```

Comment:
Yeah, incompatible change but good for hbase3 I'd say.

Reply:
If HBASE_ROOT_LOGGER defined, do you want to try parsing it and using result to populate HBASE_ROOT_LOGGER_LEVEL, etc.

Reply:
Let me try...

Reply:
Seems log4j2 also supports properties files in newer versions now. Let me see if we could use a property file so we could still use the old config pattern.